documents/src/pages/build-app/bundling-configuration.md

```diff
@@ -1,11 +1,11 @@
 <!--
 type: page
-title: Bundling Configuration
+title: Bundling Configuration Guide
 location: ./guides/bundling-configuration
 layout: default
 -->
 
-# Bundling Configuration
+# Bundling Configuration Guide
```

Comment:
Let's keep the title concise

```suggestion
# Bundling Configuration
```

Reply:
Why not use Guide after the title like Angular Guide, React Guide, Vue Guide?
https://ui.refinitiv.com/tutorials/angular
https://ui.refinitiv.com/tutorials/react
https://ui.refinitiv.com/tutorials/vue

Reply:
I find Framework name as a page title e.g. "Angular" a little confusing. Not sure what this page is about. On the other hand, "Bundling Configuration" is easy to understand.